api/app/tests/hfi/test_hfi.py

```diff
@@ -229,6 +236,31 @@ def test_valid_daily():
         assert result.valid == False
 
 
+@pytest.mark.usefixtures("mock_jwt_decode")
+def test_valid_fuel_types_response(monkeypatch):
+    """ Assert that list of FuelType objects is converted to FuelTypesResponse object correctly """
+    def mock_get_fuel_types(*args, **kwargs):
+        fuel_type_1 = FuelType(id=1, abbrev="T1", fuel_type_code="T1", description="blah",
+                               percentage_conifer=0, percentage_dead_fir=0)
+        fuel_type_2 = FuelType(id=2, abbrev="T2", fuel_type_code="T2", description="bleep",
+                               percentage_conifer=0, percentage_dead_fir=0)
+        fuel_type_3 = FuelType(id=3, abbrev="T3", fuel_type_code="T3", description="bloop",
+                               percentage_conifer=0, percentage_dead_fir=0)
+        return [fuel_type_1, fuel_type_2, fuel_type_3]
+
+    monkeypatch.setattr(app.routers.hfi_calc, 'crud_get_fuel_types', mock_get_fuel_types)
+    correct_response_file = os.path.join(os.path.dirname(__file__), 'test_valid_fuel_types_response.json')
+
+    client = TestClient(starlette_app)
+    response = client.get('/api/hfi-calc/fuel_types')
+    assert response.status_code == 200
+    assert response.headers['content-type'] == 'application/json'
+    with open(correct_response_file) as file_reader:
+        correct_response = file_reader.read()
+        correct_response = correct_response.replace('\n', '').replace(' ', '')
```

Comment:
It's better to not do a text comparison, then you don't have to do these adjustments. Rather load from file using `json.load` and then you can compare it to `response['response'].json()`